cookbooks/aws-parallelcluster-install/resources/ec2_udev_rules/ec2_udev_rules_redhat8.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+provides :ec2_udev_rules, platform: 'redhat' do |node|
+  node['platform_version'].to_i == 8
+end
+
+unified_mode true
+use 'partial/_common_udev_configuration'
+
+default_action :setup
+
+action :setup do
+  directory '/etc/udev/rules.d' do
```

Comment:
I would move this code to create_common_udev_files. That way, it would work for any OS not having the directory, while if the directory exists the block would be ignored.